src/evaluation/show.jl

```diff
@@ -39,22 +44,27 @@ function rank_table(df::DataFrame, metric_col=:tst_auc)
 	rt = vcat(results...)
 	sort!(rt, :dataset)
 	
-	# add average std
-	std_column = String(metric_col)*"_std"
-	# order must be the same as in the rt dataset
-	mean_std = [
-		mean(
-			filter(y -> ~isnan(y), 
-				filter(x -> (x.modelname == m), df)[std_column])) for m in names(rt[:,2:end])]
+	# add average std of the winning hyperparameter
+	mean_std = [mean(
+		filter(x -> (x.modelname == m), df)[std_column]) for m in names(rt[:,2:end])]
 	push!(rt, ["MEAN_STD", mean_std...])
 
+	# add top 10 std (more precisely mean over dataset)
+	top_10_std = [mean(
+		filter(x -> (x.modelname == m), df)[top_10_std_column]) for m in names(rt[:,2:end])]
+	push!(rt, ["MEAN_TOP10_STD", top_10_std...])
+	
+	# round all values in the table before computing rank
+	rt[:, 2:end] .= round.(rt[:, 2:end], digits=2)
+	
 	# add average rank
 	mask_nan_max = (x) -> (isnan(x) ? -Inf : x)
 	rs = zeros(size(rt, 2) - 1)
-	for row in eachrow(rt)
+	for row in eachrow(rt[1:end-2,:])
```

Comment:
This formerly affected the final ranking, as it took into account the `std` rows as another dataset. It would be better if in the future these two dataframes were separated.